app/Console/Commands.php

```diff
@@ -83,10 +34,9 @@ public static function latexToPdf($path, $outputDir)
         if (self::isDebugMode()) {
             $result = "ok";
         } else {
-            $command = config('commands.pdflatex') . " " . "-interaction=nonstopmode -output-dir " . $outputDir . " " . $path . " 2>&1";
-            Log::info($command);
-            $result = exec($command);
-            Log::info($result);
+            $process = new Process(['pdflatex', '-interaction=nonstopmode', '-output-dir', $outputDir, $path]);
```

Comment:
As far as I remember, the `pdflatex` command did not work inside the container in the case of the php interpreter, that is why `config('command.pdlatex')` was used.